.github/workflows/reusable/observability-health-report.yml

```
@@ -0,0 +1,56 @@
+outputs:
+  issue-url:
+    description: URL of the created issue.
+    value: ${{ jobs.publish.outputs.issue-url }}
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    outputs:
+      issue-url: ${{ steps.report.outputs.issue-url }}
```

actionlint (1.7.12)

[error] property "issue-url" is not defined in object type {}

(expression)